mesonbuild/backend/xcodebackend.py

```diff
@@ -1580,6 +1580,7 @@ def generate_single_build_target(self, objects_dict, target_name, target) -> Non
             headerdirs = []
             bridging_header = ""
             is_swift = self.is_swift_target(target)
+            langs = set()
```

Comment:
Is there a reason you can't use `target.compilers` here? It's a dict of `str: Compiler`, where str is a language.

Reply:
It had not occurred to me at the time when I wrote this, but I can certainly do that.